examples/api/lib/widgets/async/stream_builder.0.dart

```diff
@@ -30,6 +30,7 @@ class MyStatefulWidget extends StatefulWidget {
 }
 
 class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  // ignore: no_sync_async_star
```

Comment:
btw you may want to make this flutter_ignore since eventually we're going to have a lint that tells us about non-existent lints mentioned in ignores

Reply:
Done

Reply:
This one still says "ignore" instead of "fluttter_ignore"?

Reply:
Ahh I missed this, will fix.